frontend/src/views/housing/project/ProjectView.vue

```diff
@@ -38,12 +39,14 @@ const { submissionId } = defineProps<{
 
 // Constants
 const breadcrumbHome: MenuItem = { label: 'Housing', route: RouteName.HOUSING };
```

Comment:
constant should follow proper naming convention. Please use localized text for label